analysis/druidrestoration/src/Guide.tsx

```diff
@@ -0,0 +1,237 @@
+            ok={{
+              count: modules.prematureRejuvenations.highOverhealCasts,
+              label: 'High-overheal Rejuvenations',
+            }}
+            bad={{
+              count: modules.prematureRejuvenations.earlyRefreshments,
+              label: 'Clipped duration Rejuvenations',
+            }}
+          />
+        </SubSection>
+        <SubSection title="">
+          <b>
+            <SpellLink id={SPELLS.WILD_GROWTH.id} />
+          </b>{' '}
+          is your best healing spell when multiple raiders are injured. It quickly heals a lot of
+          damage, but has a high mana cost. You need to hit at least 3 injured targets for this to
+          be mana efficient over just using <SpellLink id={SPELLS.REJUVENATION.id} />.
+          <br />
+          <br />
+          <strong>Wild Growth casts</strong>
+          <small>
+            {' '}
+            - Green is a good cast, Red was effective on fewer than three targets. A hit is
+            considered "ineffective" if over the first 3 seconds it did more than 50% overhealing.
+            Mouseover boxes for details.
+          </small>
+          {modules.wildGrowth.guideTimeline}
+        </SubSection>
+        <SubSection title="">
+          <b>
+            <SpellLink id={SPELLS.REGROWTH.id} />
+          </b>{' '}
+          is for urgent spot healing. The HoT it applies is very weak - never pre-cast Regrowth.
+          Regrowth is only efficient when its direct portion doesn't overheal. Exceptions are when
+          Regrowth is free due to <SpellLink id={SPELLS.CLEARCASTING_BUFF.id} /> or cheap due to{' '}
+          <SpellLink id={SPELLS.ABUNDANCE_TALENT.id} />
+          <br />
+          <br />
+          <strong>Regrowth casts</strong>
+          <small>
+            {' '}
+            - Green is a good cast, Red is a bad cast (at full mana cost on a high health target).
+            Mouseover boxes for details.
+          </small>
+          {modules.regrowthAndClearcasting.guideTimeline}
+        </SubSection>
+        <SubSection title="">
+          <b>
+            <SpellLink id={SPELLS.LIFEBLOOM_HOT_HEAL.id} />
+          </b>{' '}
+          can only be active on one target at time and provides similar throughput to Rejuvenation.
+          However, it causes <SpellLink id={SPELLS.CLEARCASTING_BUFF.id} /> procs and so is a big
+          benefit to your mana efficiency . It should always be active on a target - the tank is
+          usually a safe bet.
+          <br />
+          {modules.lifebloom.subStatistic()}
+        </SubSection>
+        <SubSection title="">
+          <b>
+            <SpellLink id={SPELLS.EFFLORESCENCE_CAST.id} />
+          </b>{' '}
+          is extremely mana efficient if you're good about placing it where raiders are standing.
+          Under the boss is usually a safe bet. Aim to keep it active at all times.
+          <br />
+          {modules.efflorescence.subStatistic()}
```

Comment:
i know this is taking the efflo substat, but this should really be colored differently on the guide for usability reasons. everywhere else on the guide, red = bad. but the efflo one has a reddish pink = good

Reply:
I was kinda thinking the same thing, but wasn't sure on a good solution